subsys/bluetooth/host/l2cap.c

```diff
@@ -2412,6 +2412,11 @@ int bt_l2cap_chan_recv_complete(struct bt_l2cap_chan *chan, struct net_buf *buf)
 		return -ENOTCONN;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_CLASSIC) &&
+		conn->type == BT_CONN_TYPE_BR) {
```

Comment:
Incorrect split line alignment, however it also looks like the line split isn't needed at all.

Reply:
Updated.